Source/NSDockTile.m

```diff
@@ -97,7 +108,19 @@ - (NSString *) badgeLabel
 
 - (void) setBadgeLabel: (NSString *)label
 {
+  NSImage *tempImage;
+  NSImageRep *rep;
+
   ASSIGNCOPY(_badgeLabel, label);
+
+  tempImage = [[NSImage alloc] initWithSize:[_appIconImage size]];
```

Comment:
Here and in the following lines a few spaces are missing after the colons.

Reply:
no problem...